README.md

```diff
@@ -132,7 +137,6 @@ Example commands:
 
 - [ ] **[Desktop verson]**
 
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] OPTIONAL
Kindly endeavor to fill the FAQ section with at least 2 Q&A(s), on _line 174_ of your ReadMe.